### ISSUE TITLE
Warning: Accessing PropTypes via the main React package is deprecated. Use the prop-types package from npm instead.

### ISSUE THREAD

**Issue by radicalsauce** [jorosz/react-rater-plus#1]
Nice module! This is exactly the variation of react-rater I was looking for, so thanks!

FYI: I'm getting the following warning:

```
Warning: Accessing PropTypes via the main React package is deprecated. Use the prop-types package from npm instead.
```

```
Warning: Portal: React.createClass is deprecated and will be removed in version 16. Use plain JavaScript classes instead. If you're not yet ready to migrate, create-react-class is available on npm as a drop-in replacement.
```

Any chance you could remove this dependency in the next version?

**Comment by jorosz** [comment]
Hi Kelly,
thanks for letting me know!

Looks like you are using React 15.5 it has new syntax for PropTypes. I have published version 0.2.0 that uses the 'prop-types' library so no warnings should come from the Rater. React.createClass isn't used in the code.

I still get the same warning from 'react-css-themr'. It's declared as a peer dependency so you should be able to apply a newer version when it's published.

**Comment by radicalsauce** [comment]
Right on. Thanks for getting back to me!
